R/fontset-dejavu.R

```diff
@@ -0,0 +1,54 @@
+#' @references \url{http://dejavu-fonts.org}
+#' @include fontset.R
+fontset_dejavu <- function(variant = "sans", style = "book") {
+  if (system.file("fonts", package = "fontHeavy") == "") {
```

Comment:
this is the Github package that holds fonts too heavy for CRAN, so a generic name is probably better since it can potentially hold multiple fonts.